src/provider/flagsmith/flagsmith_provider.py

```diff
@@ -0,0 +1,83 @@
+import typing
```

Comment:
Thanks for this @toddbaert. I just discussed it with @ajhelsby. Is the expectation therefore that the build process (perhaps via a github workflow) would pull in this python-contrib repo and build as a single package to pypi still? 

We were thinking it would be quite cool to use the pypi subpackages functionality similar to e.g. boto so you'd end up with something like: 

```
pip install openfeature[flagsmith]
```

Reply:
Sorry for missing this for so long...

I'm the farthest thing from a python expert, but this seems like a reasonable solution. The goal is not to pull in deps for things you aren't using, and to even allow 3rd party libraries to include OpenFeature in their libraries, which the first party application could then leverage.

We basically want to package the evaluation separate from any provider implementation.